doc/classes/DampedSpringJoint2D.xml

```diff
@@ -78,12 +78,16 @@
 	</methods>
 	<members>
 		<member name="damping" type="float" setter="set_damping" getter="get_damping">
+			The spring joint's damping ratio. A value of 0 indicates an undamped spring, while 1 causes the system to reach equilibrium as fast as possible (critical damping).
```

Comment:
My other comment still applies: there needs to be an explanation of what an "undamped spring" is, something along the lines of "the higher the value, the faster the spring reaches equilibrium."

Reply:
oh ok